### PR TITLE
💄 Live page | Fix red-coloured text

### LINKED ISSUE #1831: 💄 Live page - Fix red-coloured text feature
**Describe the bug**
On https://www.ssw.com.au/live
The red coloured text in the title is created with the bold option in Tina.

**To Reproduce**
1. Go to https://www.ssw.com.au/admin/index.html#/~/live 
2. In the Tine left menu, select "Live - pages"
3. Observe that the bold text is turned into red

**Expected behavior**
The user should use CSS to turn text into red: < span class="text-sswRed" >Example< /span >

**Screenshots/Video**

Issue001b
**Figure**: Feature to turn text colour into red

### PR DESCRIPTION
**Pull request by jeoffreyfischer**
Changed the feature to turn the text colour to red.
Before: bold text was turned into red colour.
After: html and css commands can be used to change the text colour.

- Affected routes: /live

- Fixed #1831

- [ ] Include done video or screenshots

website-PR-008a
**Figure**: The user can change the text colour using html and css

### CODE REVIEW

**Review by Harry-Ross — APPROVED** [review]
LGTM ✈️

File: components/live/header.tsx [hunk 1]
```diff
@@ -26,8 +28,13 @@ export const LiveHeader = (props: LiveHeaderprops) => {
               "flex flex-row items-center max-md:justify-center"
             )}
           ></div>
+          <h1
+            dangerouslySetInnerHTML={{
+              __html: sanitiseXSS(title, spanWhitelist) || "",
+            }}
+          ></h1>
```
Comment (Harry-Ross) [inline components/live/header.tsx]: Can you make this bold like the old one, and the same size?

File: components/live/header.tsx [hunk 2]
```diff
@@ -26,8 +28,13 @@ export const LiveHeader = (props: LiveHeaderprops) => {
...
           <div className="mb-2 pb-1 pt-3 text-5xl child-h1:text-5xl child-h1:font-semibold child-p:text-xl descendant-strong:font-semibold descendant-strong:text-sswRed">
```
Comment (Harry-Ross) [inline components/live/header.tsx]: Can you remove the classes that still make it bold? Also copy all the child-h1 stuff to the className of the h1 above (without the child-h1: as a prefix)

**Review by Harry-Ross — CHANGES_REQUESTED** [review]

File: components/live/header.tsx (hunk 1 above)
Comment (jeoffreyfischer) [reply components/live/header.tsx]: Done

File: components/live/header.tsx (hunk 2 above)
Comment (jeoffreyfischer) [reply components/live/header.tsx]: Done

**Review by Harry-Ross — APPROVED** [review]

**Pull request merged**